### DISCUSSION TITLE
How to use webhook response instance

### DISCUSSION THREAD

**Discussion by dontKnew** [netflie/whatsapp-cloud-api#180] [Q&A]
The Webhook::read function will return a Notification instance. Please, [explore](https://github.com/netflie/whatsapp-cloud-api/tree/main/src/WebHook/Notification) the different notifications availables. 

I am getting the instance , but those properties are private, i can access directly, So how can i access that properties.. ??

**Comment by derrickobedgiu1** [comment]
There are public methods provided to access the entries. Follow the location where those properties have been declared and you'll find the methods to use. Some examples for Text instance:
```php
$message_text = $webhook->message();
$userid = $webhook->customer()->id();
$user_name = $webhook->customer()->name();
$messageid = $webhook->id();
```
